app/views/pages/enrolment/primary-certificate.html.erb

```diff
@@ -5,104 +5,51 @@
-                  ) %>
-                </li>
-              <% end %>
-              <li><%= t('.programme_list_2') %></li>
-              <li><%= t('.programme_list_3') %></li>
-            </ul>
-
-            <%= render DocumentCardsComponent.new(
-              class_name: 'pathway-cards',
-              show_border: false,
-              cards_per_row: 1,
-              tracking_category: 'Primary unenrolled',
-              cards: [
-                {
-                  class_name: 'developing-in-the-classroom',
-                  title_link: {
-                    title: t('.developing_title'),
-                    url: 'https://static.teachcomputing.org/primary-pathways/Developing-in-the-Classroom.pdf',
-                    tracking_label: 'Classroom PDF',
-                    file_type: '(PDF)'
-                  },
-                  body: {
-                    text: t('.developing_text')
-                  },
-                  button: {
-                    title: t('.enrol'),
-                    url: enrol_primary_certificate_path(user_programme_enrolment: { user_id: current_user&.id, programme_id: @programme.id, pathway_slug: 'developing-in-the-classroom' }),
-                    method: :post,
-                    tracking_label: 'Classroom enrol'
-                  }
-                },
-                {
-                  class_name: 'specialising-or-leading',
-                  title_link: {
-                    title: t('.specialising_title'),
-                    url: 'https://static.teachcomputing.org/primary-pathways/Specialising-or-leading.pdf',
-                    tracking_label: 'Leading PDF',
-                    file_type: '(PDF)'
-                  },
-                  body: {
-                    text: t('.specialising_text')
-                  },
-                  button: {
-                    title: t('.enrol'),
-                    url: enrol_primary_certificate_path(user_programme_enrolment: { user_id: current_user&.id, programme_id: @programme.id, pathway_slug: 'specialising-or-leading' }),
-                    method: :post,
-                    tracking_label: 'Leading enrol'
-                  },
-                }
-              ]
-            ) %>
-          </div>
+          <h2 class="govuk-heading-m"><%= t('.how_does_it_work') %></h2>
+          <p class="govuk-body"><b><%= t(
+            '.how_does_it_work_subheading',
+          ) %></b></p>
+          <ul class="govuk-list govuk-list--bullet">
+            <li>
+              <%= t(
+                    '.how_does_it_work_1',
+                    create_an_account: link_to('Create an account', '#'),
+                    log_in: link_to('log in', '#')
+              ).html_safe %>
+            </li>
+            <li><%= t('.how_does_it_work_2') %></li>
+            <li><%= t('.how_does_it_work_3') %></li>
+            <li><%= t('.how_does_it_work_4') %></li>
```

Comment:
This style is fine, but it might save you some typing in future if you recall that YAML supports arrays (known as 'sequences' apparently https://yaml.org/spec/1.2.2/#block-sequences ). Don't change what you've made so far!

Reply:
I've changed over the other lists. Didn't realise that you could get an array out of the i18n method! The "how_does_it_work" items need some links substituted in so don't really work with iterating over an array/